docs/guides/removing-props-from-html.md

```diff
@@ -31,12 +31,18 @@ const Box = styled('div', {
 
 ## Styled Components
 
-Unfortunately, Styled Components does not currently support an API to control which props are forwarded to the HTML element.
-**If you'd like to see support for this, please leave a comment on their long-running issue:**
+As of [release v5.1.0](https://styled-components.com/releases#v5.1.0) Styled Components has added a `shouldForwardProp` API. It isn't all that different from what Emotion is doing.
```

Comment:
Seems legit. Would be nice to have `shouldForwardProp` have docs to link to, but I think that can be added later, especially since you're linking to the release.